internal/apiserver/route_post_new_contract_api.go

```diff
@@ -39,7 +39,7 @@ var postNewContractAPI = &oapispec.Route{
 	FilterFactory:   nil,
 	Description:     i18n.MsgTBD,
 	JSONInputValue:  func() interface{} { return &fftypes.ContractAPI{} },
-	JSONInputMask:   []string{"ID", "Message", "Namespace"},
+	JSONInputMask:   []string{"ID", "Message", "Namespace", "URLs"},
```

Comment:
Not specifically related to this code change, but are these really `URLs`, or are they `paths`?
My understanding is these will be registered as a sub-path under a URL that has a fixed front:
`https://localhost:5000/api/v1/namespaces/default/apis/{THIS BIT}`

Reply:
These are URLs, returned from all GET operations so that you can easily find the Swagger schema and Swagger UI. Example body:
```
{
        "id": "1ca1ec36-4e01-4708-93e3-418ac926d122",
        "namespace": "default",
        "interface": {
            "id": "f894bd6b-739b-4785-9d58-59c4aa37fdac"
        },
        "ledger": null,
        "location": {
            "address": "0xb2449dc09c6d902cdca66d72057b44b32380b926"
        },
        "name": "simple-storage",
        "message": "277743ee-5a56-4955-aae5-18f5a6c04874",
        "urls": {
            "openapi": "http://127.0.0.1:5000/api/v1/namespaces/default/apis/simple-storage/api/swagger.json",
            "ui": "http://127.0.0.1:5000/api/v1/namespaces/default/apis/simple-storage/api"
        }
    }
```
Open to input on the naming and organization here, of course.